specs/span-batches.md

```diff
@@ -304,6 +312,17 @@ Span-batch rules, in validation order:
     that is invalid or derived by other means exclusively:
     - any transaction that is empty (zero length `tx_data`)
     - any [deposited transactions][g-deposit-tx-type] (identified by the transaction type prefix byte in `tx_data`)
+- Overlapped blocks checks:
+  - Note: If the span batch overlaps the current L2 safe chain, we must validate all overlapped blocks.
+  - Variables:
+    - `block_input`: an L2 block derived from the span-batch.
+    - `safe_block`: an L2 block from the current L2 safe chain, at same timestamp as `block_input`
+  - Rules:
+    - For each `block_input`, whose timestamp is less than `next_timestamp`:
+      - If there's no `safe_block` for the `block_input` -> `drop`: i.e. misaligned timestamp
```

Comment:
We don't seem to implement this check in https://github.com/ethereum-optimism/optimism/pull/7621/ but if the prev l2 block exists then the first block must have its timestamp aligned and every other block timestamp is derived by adding the block time so they must also be aligned.

Reply:
@ajsutton you're right. I think this rule is redundant and we can remove it from specs. Thank you!